unittest/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# Add test that depends on the eigenpy library
+# import eigenpy is not authorized in those tests
+function(eigenpy_add_test name)
+  jrl_boostpy_add_module(${name} ${name}.cpp)
+  target_link_libraries(${name} PRIVATE eigenpy)
+  jrl_target_set_default_compile_options(${name} PRIVATE)
+  jrl_target_enforce_msvc_conformance(${name} PRIVATE)
+
+  jrl_target_set_output_directory(${name} OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${name})
+
+  jrl_python_generate_init_py(
+      ${name}
+      OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/${name}/__init__.py
+  )
+
+  add_test(
+    NAME "Eigenpy: import ${name}"
+    COMMAND $<TARGET_FILE:Python::Interpreter> -c "import ${name}"
+  )
+
+  set(test_name "Eigenpy: test_${name}.py")
+  add_test(
+    NAME ${test_name}
+    COMMAND $<TARGET_FILE:Python::Interpreter> ${CMAKE_CURRENT_SOURCE_DIR}/test_${name}.py
+  )
+  set_property(
+    TEST ${test_name}
+    PROPERTY
+      ENVIRONMENT_MODIFICATION
+        "PYTHONPATH=path_list_prepend:$<TARGET_FILE_DIR:eigenpy>/.."
```

Comment:
I don't think this one is useful.
This give the `build/lib/..` directory where no imported lib can be found.